build.sbt

```diff
@@ -704,6 +704,14 @@ lazy val `scio-test` = project
       "org.scalactic" %% "scalactic" % scalatestVersion,
       "org.scalatest" %% "scalatest" % scalatestVersion,
       "org.typelevel" %% "cats-kernel" % catsVersion,
+      // provided
+      "com.spotify" %% "magnolify-parquet" % magnolifyVersion % Provided,
```

Comment:
I'm assuming that anyone using these helpers already has compile-time Parquet dependencies... 🤷‍♀️